mesonbuild/mtest.py

```diff
@@ -1509,6 +1509,7 @@ def __init__(self, options: argparse.Namespace):
         self.console_logger = ConsoleLogger()
         self.loggers.append(self.console_logger)
         self.need_console = False
+        self.ninja = None # type: T.List[str]
```

Comment:
You can just do `self.ninja: T.Optional[T.List[str]] = None`. No point in using a comment. I am pretty sure these type comments are deprecated some Python type checkers.

Reply:
I just tried to stay consistent with the surrounding lines...